.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
```

Comment:
@MehwishDogar `.gitignore` everything under the `.vscode` hidden directory. and before doing git ignore. delete this launch.json file and commit/push the deleted change to remove it from here as well. as a consequence of deleting the file you might end up losing/redoing some of editor configurations mentioned in this file.

Reply:
@MehwishDogar this file is not removed. please follow the steps above.